ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -107,3 +107,36 @@
     SOCIAL_AUTH_STORAGE = "ansible_base.authentication.social_auth.AuthenticatorStorage"
     SOCIAL_AUTH_STRATEGY = "ansible_base.authentication.social_auth.AuthenticatorStrategy"
     SOCIAL_AUTH_LOGIN_REDIRECT_URL = "/"
+
+
+if 'ansible_base.rbac' in INSTALLED_APPS:
+    # Settings for the RBAC system, override as necessary in app
+    ANSIBLE_BASE_ROLE_PRECREATE = {
```

Comment:
Can you document what all of these settings do? Settings like `ANSIBLE_BASE_TEAM_TEAM_ALLOWED` are very confusing.

Reply:
Endorsed.

Reply:
An issue about how to handle the lot of these https://github.com/ansible/django-ansible-base/issues/149